packages/ssx-sdk/src/ssx.ts

```diff
@@ -90,7 +90,10 @@ export class SSX {
   async resolveEns(
     /** User address */
     address: string,
-    resolveEnsOpts?: SSXEnsResolveOptions 
+    resolveEnsOpts: SSXEnsResolveOptions = {
```

Comment:
This should be partial? In case someone passes something like `resolveEns('0xabc`, { domain: true })`

Reply:
```suggestion
    resolveEnsOpts: Partial<SSXEnsResolveOptions> = {
```

Reply:
It is already partial because the interface props are optional 
```
/** ENS options supported by SSX. */
export interface SSXEnsResolveOptions {
    /** Enable ENS name/domain resolution. */
    domain?: boolean;
    /** Enable ENS avatar resolution. */
    avatar?: boolean;
}
```

And it has a default value to allow the user to only call `ssx.resolveEns("address")` and resolve both domain and avatar.

Reply:
Ah got it!